CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 # Changelog
 
-## Next version
+## Version 1.4.0 - Feature release
```

Comment:
```suggestion
## Next version
```

Reply:
then, we usually label the next version as "next" and don't specify the version number until we have the complete content. At release time, we decide whether we bump major, minor or patch